products/sle/main.pm

```diff
@@ -804,8 +804,11 @@ sub load_consoletests {
     loadtest "console/curl_https";
     if (check_var_array('SCC_ADDONS', 'asmm')) {
         loadtest "console/puppet";
+    }
+    if (check_var_array('SCC_ADDONS', 'asmm') || sle_version_at_least('15')) {
         loadtest "console/salt";
     }
+
```

Comment:
please delete blank line. We don't have this as a separator in the other places around so we should not have it here

Reply:
blank line got removed now